### DISCUSSION TITLE
Is "deepfashion2_yolov8s-seg.pt" usable in adetailer?

### DISCUSSION THREAD

**Discussion by brknsoul** [Bing-su/adetailer#341] [Q&A]
And if so, how would I use it?

**Comment by BoxJockey** [comment]
Yes, you can download the it here:
[https://huggingface.co/Bingsu/adetailer/tree/main]([https://huggingface.co/Bingsu/adetailer/tree/main]
)
Put it here: stable-diffusion-webui\models\adetailer

Does that help?
